config.py

```diff
@@ -5,18 +5,34 @@
 
 @dataclass
 class DataSourceConfig:
-    ####################
-    # DAG placeholders #
+    """
+    Configuration for a data source.
+
+    Attributes:
+        name (str): Unique name of the data source.
+        tmp_folder (str): Local path folder for storing temporary and intermediate files.
+        minio_path (str): Folder name in MinIO where files are stored.
+        file_name (str): Name of the main file associated with the data source.
+        files_to_download (dict[str, dict[str, str]]): Information about files to download.
+            Keys represent unique file identifiers, and values are dictionaries with details about the files:
+            - url (str): url of the file to download
```

Comment:
Also optional, right ?

Reply:
True, everything corrected! Thanks for the review!